tutorials/09-variational-inference/Project.toml

```diff
@@ -1,12 +1,14 @@
 [deps]
 Bijectors = "76274a88-744f-5084-9051-94815aaf08c4"
 ComponentArrays = "b0b7db55-cfe3-40fc-9ded-d10e2dbeff66"
+ConjugatePriors = "1624bea9-42b1-5fc1-afd3-e96f729c8d6c"
```

Comment:
I very strongly suggest not re-introducing ConjugatePriors. The package has not been updated in a while, does not support Julia 1.6 properly, and will cause major downgrades of Distributions and Turing. It's also not needed since all the analytical calculations can be performed in ~5 lines of code. Therefore I removed it in a PR last week.
```suggestion
```

Reply:
@devmotion Many thanks for letting me know this. I inherited it from the old version, and found some distribution methods are from it. I am looking into it.

Reply:
@devmotion Thanks for removing the `ConjugatePrior` and re-wrote the following lines:
```
p_μ = μₙ + sqrt(βₙ / (αₙ * κₙ)) * TDist(2 * αₙ)
p_μ_pdf = z -> pdf(p_μ, z)
```
I guess in the first line you want to do the linear transform `y=a+b*x` (correct me if I am wrong), which induces the current building failure `No method matching *(float64, Distribution)` - for some reason it cannot build a linear transform function `p_μ(x)` by scaling a distribution (even though we take the sampling view). I therefore tried re-writing it in the following form, which avoids the building error.
```
p_μ0 = TDist(2 * αₙ)
p_μ_pdf = z -> pdf(p_μ0, (z-μₙ)/sqrt(βₙ / (αₙ * κₙ)))/sqrt(βₙ / (αₙ * κₙ))
```
This is constructed following the principle of probability of transformed variables:
f_Y(y) = 1/b * f_X((y-a)/b)
Not sure if my understanding is correct; it would be helpful if you can confirm it. Many thanks.

Reply:
Just revert your latest changes. The current code works just fine *if* you use a somewhat recent version of Distributions. As I said, re-introducing ConjugatePriors downgrades Distributions *and* Turing a lot.

So my suggestion is:
- reverting the latest commit with your rewrites
- updating the Manifest.toml file by running `] up`

Reply:
Great, `ConjugatePriors` was removed and `p_μ_pdf ` reversed.